### ISSUE TITLE
Update the README with the goals and guiding strategy of this website

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/2i2c-brochure#111]
### Context

This website is one of the most prominent public-facing parts of 2i2c, and it's important that we clearly define our goals for the site, and our strategy for how to use it to accomplish our goals. This currently is not defined anywhere in an explicit way, and doing so will make it easier to make decisions about how to change the content and structure.

### Proposal

We should define a few major goals for this site and display them prominently in its README (above the sections about how to do various things with it). Here are a few suggested goals:

- Answer "what is 2i2c and why does it exist?" in a **glanceable** way. This includes our mission, major strategy, and major operational efforts.
- Make it easy to understand without a lot of context for the key stakeholders we care about
- Demonstrate the _impact_ of 2i2c's operations
- Build trust with communities that don't know us by demonstrating our work with others
- Provide **calls to action** according to next steps we want others to take
- Provide links to deeper material for people that want to learn more

**Comment by sgibson91** [comment]
Why put this info in the README and not on the website itself? Or is it already on the website and this is a recap with links to the appropriate sections on the website?

**Comment by choldgraf** [comment]
My thinking was because the README is meant for _internal team members_ who work on the website, whereas the website itself is meant for external stakeholders.

**Comment by sgibson91** [comment]
Yeah I agree, and all the info you've listed seems great stuff to put on the website too? I'd want to know all that stuff if I was an external stakeholder too, I think.

**Comment by choldgraf** [comment]
Where would you suggest placing it within the website?

**Comment by sgibson91** [comment]
Front page 😄 In shorthand format, then links to deeper dives as you've suggested. This all feels like really important info that anyone wanting to work with us would want to know about, and I guess I'm not quite understanding the internal/external divide you've drawn.

As an internal team member, I want to know how to build the site locally and maybe also the structure of the repo so I can figure out where my contribution best lives. I can actually visit the site to learn about the content.

I'm not sure if I'm making sense though, or have just misunderstood.

**Comment by sgibson91** [comment]
Oh ok upon re-reading, it sounds like your bullet list is a copy-pastable example of _exactly what_ we should put in the README, and _are not_ questions to answer, write up, and _then_ post on the website.

Ok that seems reasonable, I think I'm onboard now.
